src/stores/printing.js

```diff
@@ -90,5 +100,20 @@ export const usePrintingStore = defineStore('printing', {
       }
       return { success, data, errors }
     },
+
+    /**
+     * fetches the list of workflows from traction
+     * @returns {Object}  {success: Boolean, data: Object, errors: Array} Is the fetch successful? The data and any errors
+     */
+    async fetchWorkflows() {
+      const rootStore = useRootStore()
+      const request = rootStore.api.v1.traction.workflows
```

Comment:
Also, we are currently in the middle of migrating from Axios to Fetch, where api.v1 uses Axios and api.v2 uses Fetch. Therefore, it might be better to use api.v2 for this, since it is a new addition.